config.sub

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
Is it possible to do this inline in `configure` without a separate file?

Reply:
Yes, but config.sub is traditional, and if somebody needs one of the arches not yet in this config.sub,
they can drop in the gnu one and get unblocked.  So there's utility in keeping it a separate file.

Reply:
Is it possible to merge this file into the configure script?

Reply:
Is it possible to merge this file into the configure script?